content-src/components/CollapsibleSection/CollapsibleSection.jsx

```diff
@@ -171,10 +178,8 @@ export class _CollapsibleSection extends React.PureComponent {
               </span>
               <span className="learn-more-link-wrapper">
                 {learnMore &&
-                  <span className="learn-more-link">
-                    <a href={learnMore.link.href}>
-                      <FormattedMessage id={learnMore.link.id} />
-                    </a>
+                  <span className="learn-more-link" data-l10n-id={learnMore.link.id}>
+                    <a data-l10n-name="learn-more" href={learnMore.link.href} />
```

Comment:
Sorry for the late comment, this doesn't seem right ? If the whole string is a link, then the html doesn't need to be exposed to localizers. So put the `data-l10n-id` on the `<a>` like what you did for "more-recommendations"